### PR TITLE
BS-13526 Prompt warning when closing studio

### PR DESCRIPTION
**Pull request by rbioteau**
English reviewed

### CODE REVIEW

File: bundles/plugins/org.bonitasoft.studio.application/messages.properties [hunk 1]
```diff
@@ -29,4 +29,5 @@ noProcessAvailable=No diagram opened recently
 offlineRepositoryTitle=Offline repository 
 offlineRepositoryMessage={0} repository is currently offline. You won't be able to update/commit changes in offline mode.
 incompatibleJavaVersionTitle=Incompatible java runtime environment version
-incompatibleJavaVersionMessage=You are running {0} using an incompatible JRE version ({1}).\nPlease refer to the <a href="http://documentation.bonitasoft.com/hardware-and-software-requirements">online documentation</a> to check java requirements.
\ No newline at end of file
+incompatibleJavaVersionMessage=You are running {0} using an incompatible JRE version ({1}).\nPlease refer to the <a href="http://documentation.bonitasoft.com/hardware-and-software-requirements">online documentation</a> to check java requirements.
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.application/messages.properties]: shouldn't we use a redirect url?

Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.application/messages.properties]: I don't known. Not part of the pr anyway...
